aws/table_aws_glue_ml_transform.go

```diff
@@ -0,0 +1,438 @@
+				Hydrate:     getGlueMLTransform,
+			},
+			{
+				Name:        "label_count",
+				Description: "The number of labels available for this transform.",
+				Type:        proto.ColumnType_INT,
+				Hydrate:     getGlueMLTransform,
+			},
+			{
+				Name:        "max_capacity",
+				Description: "The number of Glue data processing units (DPUs) that are allocated to task runs for this transform.",
+				Type:        proto.ColumnType_DOUBLE,
+				Hydrate:     getGlueMLTransform,
+			},
+			{
+				Name:        "max_retries",
+				Description: "The maximum number of times to retry a task for this transform after a task run fails.",
+				Type:        proto.ColumnType_INT,
+				Hydrate:     getGlueMLTransform,
+			},
+			{
+				Name:        "number_of_workers",
+				Description: "The number of workers of a defined workerType that are allocated when this task runs.",
+				Type:        proto.ColumnType_INT,
+				Hydrate:     getGlueMLTransform,
+			},
+			{
+				Name:        "worker_type",
+				Description: "The type of predefined worker that is allocated when this task runs. Accepts a value of Standard, G.1X, or G.2X.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getGlueMLTransform,
+			},
+			{
+				Name:        "transform_type",
+				Description: "The type of machine learning transform.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getGlueMLTransform,
+				Transform:   transform.FromField("Parameters.TransformType"),
+			},
+			{
+				Name:        "timeout",
+				Description: "The timeout for a task run for this transform in minutes.",
+				Type:        proto.ColumnType_INT,
+				Hydrate:     getGlueMLTransform,
+			},
+			{
+				Name:        "role",
+				Description: "The name or Amazon Resource Name (ARN) of the IAM role with the required permissions.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getGlueMLTransform,
+			},
+			{
+				Name:        "input_record_tables",
+				Description: "A list of Glue table definitions used by the transform.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getGlueMLTransform,
+			},
+			{
+				Name:        "parameters",
+				Description: "The configuration parameters that are specific to the algorithm used.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getGlueMLTransform,
+			},
+			{
+				Name:        "schema",
+				Description: "The Map object that represents the schema that this transform accepts.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getGlueMLTransform,
+			},
+			{
+				Name:        "evaluation_metrics",
+				Description: "The latest evaluation metrics.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getGlueMLTransform,
+			},
+			{
+				Name:        "transform_encryption",
+				Description: "The encryption-at-rest settings of the transform that apply to accessing user data.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getGlueMLTransform,
+			},
+
```

Comment:
@ParthaI is the schema missing the `tags_src` column?

Reply:
It is not necessary to add the `tags_src` column for this table , as the API response is of type `map[string]string`.